bin/refresh-open-pulls

```diff
@@ -5,6 +5,6 @@ var refresh = require('../lib/refresh');
 var db = require('../lib/db');
 
 refresh.openPulls()
-.done(function() {
+.then(function() {
```

Comment:
Pretty sure this was `done` so that unhandled rejected promises still get reported instead of disappearing. Thoughts?